src/pages/Check.jsx

```diff
@@ -1,7 +1,14 @@
-const Check = () => (
-  <div>
-    <h1>Check</h1>
-  </div>
-)
+import { useState } from 'react'
+import Modal from '../components/Modal'
+
+const Check = () => {
+  const [modState, changeModState] = useState(true)
```

Comment:
please use full names, for example `modalState`. 

Also we can describe the way the state will be used, for example here is used to open and close the modal. As the initial state is closed we can say:
```suggestion
  const [isModalOpen, changeIsModalOpen] = useState(false)
```

Reply:
Thank you for the advice, i used more descriptive ones